be/src/services/entity.service.ts

```diff
@@ -121,6 +124,110 @@ export class EntityService {
         }
       });
   }
+    public async listEntities(query: ListQueryModel, id?: string | ObjectId): Promise<ListingResult<Entity>> {
+        const {options, filter} = this.transformToMongoDBFilterOption(query, id);
+        // @ts-ignore
+        const dbos: Entity[] = (await EntityService.collection().find(filter, options).toArray()) as Entity[];
+        return {
+            offset: query.offset,
+            rows: dbos.length,
+            totalItems: 0, // TODO
+            items: dbos
+        };
+    }
+
+    private escapeRegExp(string: string): string {
+        return string.replace(/[.*+?^${}()|[\]\\]/g, '');
+    }
+
+    // eslint-disable-next-line max-len
+    private transformToMongoDBFilterOption(query?: ListQueryModel, id?: string | ObjectId): {options: FindOptions; filter: Filter<Entity>} {
+        const options: FindOptions = {};
+        const filter: Filter<Entity> = {};
+
+        if (!!id) {
+            // @ts-ignore
+            filter.vocabulary = new ObjectId(id);
+        }
+
+        if (!!query?.type){
+            /* eslint-disable */
+            if (Object.values(TagType).includes(query.type as TagType)){
+                filter.type = query.type
+            } else{
+                throw new KnowledgeError(404, 'Bad Request', "Invalid Parameter of type 'type'!")
+            }
```

Comment:
After the if/else you should enable the eslint again, otherwise the whole below file is ignored. Please do that, you already have my approval